tsl/test/sql/compression_bgw.sql

```diff
@@ -205,11 +205,7 @@ SELECT add_compression_policy AS job_id
 -- job compresses only 1 chunk at a time --
 SELECT alter_job(id,config:=jsonb_set(config,'{maxchunks_to_compress}', '1'))
  FROM _timescaledb_config.bgw_job WHERE id = :job_id;
-SELECT alter_job(id,config:=jsonb_set(config,'{verbose_log}', 'true'))
- FROM _timescaledb_config.bgw_job WHERE id = :job_id;
-set client_min_messages TO LOG;
 CALL run_job(:job_id);
```

Comment:
We want to verify that the message is still logged when verbose_log is set in the path. The changes to .sql and .out files are not needed. Please remove them.